animesaturn_downloader.py

```diff
@@ -66,12 +86,13 @@ def main(main_url: str, ep_range_start: int, ep_range_end: int):
   for link in links:
     ep_name = link.split('/')[-1]
     ep_name_dest = os.path.join(out_dir, ep_name)
-
     ep_num = int(ep_name.split('-')[-1])
-    if ep_range_start and ep_range_end:
-      if ep_num < ep_range_start or ep_num > ep_range_end:
-        logging.info(f"Skipping {ep_name}")
-        continue
+
+    if is_episode_alredy_present(out_dir, ep_num) or (
```

Comment:
if ep_range_start and ep_range_end:
      if ep_num < ep_range_start or ep_num > ep_range_end:
          logging.info(f"Skipping {ep_name}")
          continue

if is_episode_alredy_present(out_dir, ep_num):
    logging.info(f"Skipping {ep_name}")
    continue